torch/distributed/optim/__init__.py

```diff
@@ -18,6 +18,7 @@
 from .functional_adamax import _FunctionalAdamax
 from .utils import as_functional_optim
 from .apply_optimizer_in_backward import _apply_optimizer_in_backward
+from .named_optimizer import NamedOptimizer
```

Comment:
Before this PR and can be done in a follow-up: Should we sort these imports?

Reply:
Sure, let me run the formatter